src/fs_utils.rs

```diff
@@ -17,6 +17,7 @@ pub enum ChangeKind {
     StaticFiles,
     Sass,
     Config,
+    Unknown,
```

Comment:
let's call it AdditionalDirectory or something similar, it's not strictly unknown

Reply:
I went with `ExtraPath` to match other names used in this feature and added a doc comment to `ChangeKind`. Let me know if you'd prefer something else though.